source/index.html.md

```diff
@@ -540,26 +527,103 @@ Exactly one payment method must be used.
   <dd>
     Amount in minor units of given currency (e.g. cents if in Euro).
   </dd>
+
   <dt>currency
     <span class="type">[A-Z]{3}</span>
   </dt>
   <dd>
     <a target="_blank" href="currencies.txt">3-letter currency code</a>. (Some exponents differ from ISO 4217.)
   </dd>
+
+  <dt>credential
+    <span class="type">(<code>storing</code>|<code>using</code>)</span>
+  </dt>
+  <dd>
+    Indicate if the payment credential (e.g. PAN and expiry) will be stored for
+    a later transaction where the payment credential is not provided by the
+    cardholder but collected from (encrypted) storage.
+    <br />
+    <code>storing</code>: The payment credential will be stored; may only be
+    stored if the authorization is approved.
+    <br />
+    <code>using</code>: The payment credential was already stored and are now
+    being used.
+    <br />
+    If <code>initiator</code> is <code>merchant</code> then
+    <code>storing</code> is invalid and <code>using</code> is assumed.
+    For a first-in-series authorization, <code>storing</code> is assumed.
+
+    <div class="type">Optional</code>
+  </dd>
+
+  <dt>initiator
+    <span class="type">(<code>cardholder</code>|<code>merchant</code>)</span>
+  </dt>
+  <dd>
+    The initiator of the authorization. An authorization is initiated by the
+    cardholder if the cardholder decided the transaction should be created.
+    This is regardless of whether stored payment credential is being used.
```

Comment:
```suggestion
    This is regardless of whether a stored payment credential is being used.
```

Reply:
I am in doubt if we want to word it like "using a stored credential" or "using stored credential". Looking at Visa, they use both; perhaps there's some systematism that I just don't get?

Reply:
I think my suggestion was prompted by "**the** payment credential" in many of the above lines.

I don't think there is any systematism about this in scheme documentation, however, I don't know for sure.